GameApplication/GameApplication/Views/Home/Index.cshtml

```diff
@@ -1,35 +1,4 @@
-﻿@{
-    ViewData["Title"] = "Home Page";
-}
-
-<script>
-    $(document).ready(function () {
-        startChat();
```

Comment:
I think you should call this function in GeneralChat/Index.cshtml

Reply:
Requested changes applied